image/DockerfileBase

```diff
@@ -0,0 +1,151 @@
+FROM ubuntu:latest
+
+ENV DEBIAN_FRONTEND noninteractive
+
+SHELL ["/bin/bash", "-ci"]
+
+# Get the basic stuff and create chipStarUser user with sudo privileges
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y sudo; \
+    useradd -ms /bin/bash chipStarUser && \
+    usermod -aG sudo chipStarUser; \
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# Set as default user
+USER chipStarUser
+WORKDIR /home/chipStarUser
+
+
+RUN sudo apt update; sudo apt install -y gcc g++ cmake python3 git software-properties-common; \
+    sudo add-apt-repository ppa:ocl-icd/ppa;  \
+    sudo apt update; \
+    sudo apt install -y python3-dev python3-yaml libpython3-dev build-essential cmake git pkg-config make ninja-build ocl-icd-libopencl1 ocl-icd-dev ocl-icd-opencl-dev libhwloc-dev zlib1g zlib1g-dev clinfo dialog apt-utils libxml2-dev wget lua5.3 lua-bit32:amd64 lua-posix:amd64 lua-posix-dev liblua5.3-0:amd64 liblua5.3-dev:amd64 tcl tcl-dev tcl8.6 tcl8.6-dev:amd64 libtcl8.6:amd64
+
+ADD configure_llvm.sh /home/chipStarUser/configure_llvm.sh
```

Comment:
Can we just use `.../scripts/configure_llvm.sh`? to avoid having a copy?

Reply:
Also, probably want to cleanup 

```
chipStarUser@b57ec91c4164:~$ ls
Lmod-8.7  Lmod-8.7.tar.bz2  configure_llvm.sh  llvm-project  pocl
```

Reply:
I need a custom script to install clang-tools-extra project as well for clang-tidy, it can be incorporated as another option in chipstar/scripts/configure_llvm.sh but I didn't want to touch that.

Reply:
Cleaning up can be done at the end of the base image docker file. This way it can reuse the previous layers and it will be built in no time and it will slim down the image substantialy.